pkg/util/fixedpool/fixedpool.go

```diff
@@ -2,11 +2,18 @@ package fixedpool
 
 import (
 	"sync"
+
+	"github.com/karmada-io/karmada/pkg/metrics"
+)
+
+const (
+	poolType = "fixedPool"
```

Comment:
What dose this type mean? Are there any other values of types? Is this a necessary value?

Reply:
Maybe we have anther pool type. For example we may use a `dynamic pool` in the future. It will scale the pool capacity according the history of pool size. So here use this label to distinguish.

Reply:
But the pool we're using is the `FixedPool`.

Reply:
I mean all the pool use this metric. As thus, when it changes to another type of pool, monitor server (such as grafana) don't need to change.

Reply:
How about adding this field after a new type of pool is designed?

Reply:
ok